.coveragerc

```diff
@@ -525,6 +525,7 @@ omit =
     homeassistant/components/mpd/media_player.py
     homeassistant/components/mqtt_room/sensor.py
     homeassistant/components/msteams/notify.py
+    homeassistant/components/mullvad/*
```

Comment:
Do not exclude all files for this integration. Since it uses a UI based configuration flow, tests for the `config_flow.py` are required.

Please note that the config flow requires a 100% test coverage.

Reply:
:/ I don't know how to properly test things. I'll give it my best shot, but it's over my head.